src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-roles.html

```diff
@@ -0,0 +1,142 @@
+
+
+<dom-module id="lancie-admin-roles">
+  <template>
+    <style include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        display: block;
+      }
+
+      paper-card {
+        width: calc(100% - 32px);
+        margin: 16px;
+      }
+
+      iron-icon {
+        margin-right: 10px;
+      }
+
+      [hidden] {
+        display: none !important;
+      }
+
+      @media (max-width: 640px) {
+        paper-card {
+          width: 100%;
+          margin: 0;
+        }
+      }
+
+      paper-button.green {
+        background-color: var(--paper-green-500);
+        color: white;
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="endpointAjax" refurl="users" on-lancie-ajax="onResponse">
+    </lancie-ajax>
+    <lancie-ajax id="ajaxRefresh" refurl="users" on-lancie-ajax="onResponse">
+    </lancie-ajax>
+    <lancie-ajax id="ajaxAddRole" method="POST" body='{"role": "[[selectedRole]]"}' refurl="users/[[selected.id]]/role"></lancie-ajax>
+    <lancie-ajax id="ajaxDeleteRole" method="POST" body='{"role": "[[selectedRole]]"}' refurl="users/[[selected.id]]/role/delete"></lancie-ajax>
+    
+    <paper-card heading="Roles">
+
+        <div class="card-content">
+          <p>Add roles to selected users</p>
+          <lancie-admin-table id="table" column-mappings="[[mappings]]" column-renderers="[[renderers]]"></lancie-admin-table>
+          
+          <lancie-form id="addMemberForm" refurl="web/committee" on-response="onAddResponse">
+          
+            <paper-dropdown-menu id="role" label="Roles">
+              <paper-listbox slot="dropdown-content" class="dropdown-content" on-iron-select="roleSelected">
+                <paper-item>Admin</paper-item>
+                <paper-item>Operator</paper-item>
+                <paper-item>Committee</paper-item>
+              </paper-listbox>
+            </paper-dropdown-menu>
+          </lancie-form>
+        </div>
+        <div class="card-actions">
+          <paper-button on-tap="addRoles">Add</paper-button>
+          <paper-button on-tap="deleteRoles">Delete</paper-button>
+        </div>
+    </paper-card>
+
+  </template>
+  <script>
+    (function () {
+      'use strict';
+
+      Polymer({
+        is: 'lancie-admin-roles',
+        properties: {
+          data: Object,
+          mappings: {
+            type: Object,
+            value: function() {
+              return {
+                profile: ['displayName'],
+              };
+            }
+          },
+          renderers: {
+            type: Object,
+            value: function() {
+              return {};
+            }
+          }
+        },
+
+        onResponse: function (e, request) {
+          if (request.succeeded) {
+            this.$.table.setData(request.response);
+          }
+        },
+
+        addRoles: function () {
+          var selectedGroup = this.$.table.shadowRoot.querySelector('vaadin-grid').selectedItems
```

Comment:
Didn't you create a function in the lancie-table which gets the selected items? Maybe use that in this file.